### PR TITLE
Release 1.3

### PR DESCRIPTION
**Pull request by CharliePoole**
This will be the 1.3 release after merge.

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,4 +1,60 @@
...
+
+GUI
+
+ * The runtime selection menu is now disabled when tests are being run in
+   process, since no other runtime is available within the GUI process.
+
+Engine
+
+ * The UserSettings service is no longer part of the engine but is handled
+   directly in the GUI. This is not considered a breaking change, since
```
Comment (mikkelbu) [inline CHANGES.txt]: "since since" => "since"

File: CHANGES.txt [hunk 2]
```diff
@@ -1,4 +1,60 @@
...
+   since the engine is not yet released separately from the GUI.
+
+ * A new PackageSettings service has been added to the engine. It's job
```
Comment (mikkelbu) [inline CHANGES.txt]: "It's" => "Its"

File: CHANGES.txt [hunk 3]
```diff
@@ -1,4 +1,60 @@
...
+   is to analyze all packages, examine assemblies and set default
+   parameters for executing them.
+
+ * A pair of agents are provided for .NET 4.0 in addition to the existing
+   .NET 2.0 agents. The engine is also now built for both .NET 2.0 and 4.0
```
Comment (mikkelbu) [inline CHANGES.txt]: "also now" => "now also" (I'm unsure about this)

Comment (CharliePoole) [reply CHANGES.txt]: Both are awkward... "In addition, the engine is now built for both runtimes." seems better.

**Pull request merged**